src/components/modal/components/dialogInput/DialogInput.tsx

```diff
@@ -0,0 +1,11 @@
+import "./dialogInput.css";
+
+interface DialogInputProps {
+  value?: string | undefined;
```

Comment:
여기 value는 optional props여서 이렇게만 써주셔도 될것같아요!
```suggestion
  value?: string;
```